packages/components/src/components/cell-output.tsx

```diff
@@ -1,11 +1,10 @@
 import { useState } from 'react';
-import { useSettings } from '@/components/use-settings';
 import { Ban, Maximize, Minimize, PanelBottomClose, PanelBottomOpen, Sparkles } from 'lucide-react';
 import { CodeCellType, PackageJsonCellType, TsServerDiagnosticType } from '@srcbook/shared';
-import { cn } from '@/lib/utils';
-import { Tabs, TabsList, TabsTrigger, TabsContent } from '@/components/ui/underline-flat-tabs';
-import { useCells } from '@/components/use-cell';
-import { OutputType, StdoutOutputType, StderrOutputType, CellModeType } from '@/types';
+import { cn } from '../lib/utils';
+import { Tabs, TabsList, TabsTrigger, TabsContent } from './ui/underline-flat-tabs';
+import { useCells } from './use-cell';
+import { OutputType, StdoutOutputType, StderrOutputType, CellModeType } from '../types';
```

Comment:
Note that the `@` prefix notation won't work right in child packages - that syntax is included in the bundled output and therefore is picked up by the package resolution logic in the parent package 🤦 . I couldn't figure out how to get this to work in `@srcbook/components` so I converted everything in that package over to use relative imports.

Reply:
Sounds good, I believe that is specified in the tsconfig.json `paths` field and also vite.config.ts (though I don't think this package uses vite)

Reply:
It is, yes, but that doesn't seem to convert to relative paths in the built output.